container/dockerd-entrypoint.sh

```diff
@@ -2,14 +2,25 @@
 set -e
 set -x
 
+export IFS=' '
+MMS_NUM_WORKER=1
```

Comment:
is MMS_NUM_WORKER configurable? one worker for one instance as default?

Reply:
Yes it is, but right now we are keeping it for 1 for best latency. (I replied in the morning but I could not find my reply)